### PR TITLE
SCRUM-26 Results Endpoint

### PR DESCRIPTION
**Pull request by f-hruza0111**

### CODE REVIEW

**Comment by Theanko1412** [comment]
lgtm, maybe findByUser can be named findByUserId, or use "findOptimizationResultsByUser_Id" from jpa so you dont need custom query but for MVP tommorow its fine

**Review by Theanko1412 — DISMISSED** [review]

**Review by Theanko1412 — APPROVED** [review]

File: backend/production-optimiser-api/src/main/java/it/polimi/productionoptimiserapi/controllers/ResultController.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```
Comment (giovanni-orciuolo) [inline backend/production-optimiser-api/src/main/java/it/polimi/productionoptimiserapi/controllers/ResultController.java:15]: Can add `@RequestMapping("/api/results")` here so every other mapping can be made as a subpath (e.g. `@GetMapping("/dummy")`

Comment (f-hruza0111) [reply backend/production-optimiser-api/src/main/java/it/polimi/productionoptimiserapi/controllers/ResultController.java:15]: Done

**Review by Theanko1412 — DISMISSED** [review]

**Review by Theanko1412 — APPROVED** [review]
good for now

**Pull request merged**